bigtable-hbase-1.0/pom.xml

```diff
@@ -52,6 +52,10 @@ limitations under the License.
             <groupId>org.apache.hbase</groupId>
             <artifactId>hbase-common</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Will this exclude transient dependencies, or we need to exclude them?
https://github.com/derjust/cloud-bigtable-client/commit/c775b09b7a9a64344910f17a679afbc42bccfb77

Just want to confirm.